packages/composites/proof-of-reserves/src/utils/reduce.ts

```diff
@@ -35,6 +36,7 @@ export const runReduceAdapter = async (
       return returnParsedUnits(input.jobRunID, input.data.result as string, 8)
     case lotus.name:
     case adaBalance.NAME:
+    case tokenBalance.name:
```

Comment:
We need to 

return returnParsedUnits(input.jobRunID, input.data.result as string, 18)

Since it returns 18 decimal